prow/apis/prowjobs/v1/types.go

```diff
@@ -646,6 +647,30 @@ func (d *DecorationConfig) Validate() error {
 	return nil
 }
 
+const (
+	logMountName            = "logs"
+	codeMountName           = "code"
+	toolsMountName          = "tools"
+	gcsCredentialsMountName = "gcs-credentials"
+	s3CredentialsMountName  = "s3-credentials"
```

Comment:
Why do we redefine them instead of using `prow/pod-utils/decorate.VolumeMounts()`?

Reply:
updated